literatur.bib

```diff
@@ -4308,6 +4308,16 @@ @Misc{drexler-et-al-zenodo2022b
   howpublished = "\url{https://doi.org/10.5281/zenodo.6381592}"
 }
 
+@Misc{drexler-et-al-zenodo2023b,
```

Comment:
If there's drexler-et-al-zenodo2023b here, we should have drexler-et-al-zenodo2023**a** also here.